app/helpers/application_helper/toolbar/physical_server_center.rb

```diff
@@ -0,0 +1,257 @@
+                       N_('Introspect Nodes'),
+                       :url_parms => "main_div",
+                       :confirm   => N_("Introspect selected items?"),
+                       :onwhen    => "1+",
+                       :klass     => ApplicationHelper::Button::HostIntrospectProvide
+                     ),
+                     button(
+                       :host_provide,
+                       'pficon pficon-edit fa-lg',
+                       N_('Provide selected items'),
+                       N_('Provide Nodes'),
+                       :url_parms => "main_div",
+                       :confirm   => N_("Provide selected items?"),
+                       :onwhen    => "1+",
+                       :klass     => ApplicationHelper::Button::HostIntrospectProvide
+                     ),
+                     button(
+                       :host_discover,
+                       'fa fa-search fa-lg',
+                       t = N_('Discover items'),
+                       t,
+                       :url       => "/discover",
+                       :url_parms => "?discover_type=hosts"
+                     ),
+                     separator,
+                     button(
+                       :host_new,
+                       'pficon pficon-add-circle-o fa-lg',
+                       t = N_('Add a New item'),
+                       t,
+                       :url => "/new"
+                     ),
+                     button(
+                       :host_edit,
+                       'pficon pficon-edit fa-lg',
+                       t = N_('Edit Selected items'),
+                       t,
+                       :url_parms => "main_div",
+                       :enabled   => false,
+                       :onwhen    => "1+"
+                     ),
+                     button(
+                       :host_delete,
+                       'pficon pficon-delete fa-lg',
+                       N_('Remove Selected items'),
+                       N_('Remove items'),
+                       :url_parms => "main_div",
+                       :confirm   => N_("Warning: The selected items and ALL of their"\
+                                        " components will be permanently removed!?"),
+                       :enabled   => false,
+                       :onwhen    => "1+"
+                     ),
+                   ]
+                 ),
+               ])
+  button_group('host_policy',
+               [
+                 select(
+                   :host_policy_choice,
+                   'fa fa-shield fa-lg',
+                   t = N_('Policy'),
+                   t,
+                   :items => [
+                     button(
+                       :host_protect,
+                       'pficon pficon-edit fa-lg',
+                       N_('Manage Policies for this item'),
+                       N_('Manage Policies'),
+                       :klass => ApplicationHelper::Button::HostProtect
```

Comment:
Ok, @AparnaKarve .
This came from a copy and paste to a fastest development.
We'll discuss here about this issue, PhysicalServer can have a policy.
Probably we'll implement this for the next PR.
cc @walteraa @rodneyhbrown7 @skovic